libs/blocks/editorial-card/editorial-card.css

```diff
@@ -44,6 +44,10 @@
   box-shadow: 0 3px 6px 0  rgba(0 0 0 / 16%);
 }
 
+.editorial-card .action-area a:not([class*="button"]) {
+  font-size: var(--type-body-s-size);
+}
```

Comment:
should it be everything apart of the button specifically, or rather matching other specific elements?

Reply:
This is for the link elements that are not `*-button` class, basically Text CTAs, in the card footer